clusterloader2/pkg/measurement/common/service_creation_latency.go

```diff
@@ -147,11 +147,16 @@ func (s *serviceCreationLatencyMeasurement) start() error {
 
 func (s *serviceCreationLatencyMeasurement) waitForReady() error {
 	return wait.Poll(defaultCheckInterval, s.waitTimeout, func() (bool, error) {
-		reachable := s.creationTimes.Count(reachabilityPhase)
-		ipAssigned := s.creationTimes.Count(ipAssigningPhase)
-		created := s.creationTimes.Count(creatingPhase)
-		klog.Infof("%s: %d created, %d ipAssigned, %d reachable", s, created, ipAssigned, reachable)
-		return created == reachable, nil
+		for _, svcType := range []corev1.ServiceType{corev1.ServiceTypeClusterIP, corev1.ServiceTypeNodePort, corev1.ServiceTypeLoadBalancer} {
+			reachable := s.creationTimes.Count(phaseName(reachabilityPhase, svcType))
+			ipAssigned := s.creationTimes.Count(phaseName(ipAssigningPhase, svcType))
```

Comment:
This will break now - because it's not being set for ClusterIP and NodePort, right?

Reply:
sorry forget it, fixed!